### DISCUSSION TITLE
Master/detail (parent-child) form?

### DISCUSSION THREAD

**Discussion by catamaican** [vueform/vueform#183] [Questions]
Is it possible to create a master/detail (parent/child) forms? (e.g. class/students or order/items-on-order) where the details are in a CRUD-based table? If yes, could you, please, provide an example?

**Comment by adamberecz** [comment]
I'm not entirely sure what you mean, but this might what you're looking for:
https://vueform.com/reference/list-element

It's not an actual table but you can put multiple fields in a row and repeat them.
